en/flying/pre_flight_checks.md

```diff
@@ -48,6 +48,13 @@ The following errors (with associated checks and parameters) are reported by the
 * This check only applies to boards with more than one IMU.
 * The check is controlled by the [COM_ARM_IMU_GYR](../advanced_config/parameter_reference.md#COM_ARM_IMU_GYR) parameter.
 
+`PREFLIGHT FAIL: COMPASS SENSORS INCONSISTENT - CHECK CALIBRATION`:
+* This error message is produced when measurments from different compass sensors is too great.
+* It indicates bad calibration, orientation or magnetic interference.
+* This check only applies to when more than one compass/magnetometer is connected.
+* The check is controlled by the [COM_ARM_MAG](../advanced_config/parameter_reference.md#COM_ARM_MAG) parameter.
+* In addition to calibration, you might disable the internal compass using [CAL_MAG0_EN](https://docs.px4.io/en/advanced_config/parameter_reference.html#CAL_MAG0_EN).
```

Comment:
Is this correct? Ie is it reasonable to disable it, and is CAL_MAG0_EN always the internal compass?

Reply:
We can suggest to disable the internal mags as alternative but I would not recommend it - it can still be used as a failsafe, even if its data is not great.
It is not necessarily `CAL_MAG0_EN`, and it can also be multiple. You can look at the rotation parameters, the internal ones are marked as 'internal'.